mkdocs/docs/tutorials/deployments/slo-validation-chaos.md

```diff
@@ -1,14 +1,14 @@
-# Chaos Testing with SLO Validation
+# SLO Validation with Chaos Injection
 
 !!! tip "Scenario: Inject Chaos into Kubernetes cluster and verify if application can satisfy SLOs"
     **Problem:** You have a Kubernetes app. You want to stress test it by injecting chaos, and verify that it can satisfy service-level objectives (SLOs). This helps you guarantee that your application is resilient, and works well even under periods of stress (like intermittent pod failures).
 
-    **Solution:** You will launch a Kubernetes application along with a composite experiment, consisting of [Litmus Chaos](https://litmuschaos.io/) experiment resource and an Iter8 experiment resource. The chaos experiment will delete pods of the application periodically, while the Iter8 experiment will send requests to the application and verify if it is able to satisfy SLOs.
+    **Solution:** You will launch a Kubernetes application along with a Helm chart consisting of a [Litmus Chaos](https://litmuschaos.io/) experiment, and an Iter8 experiment. The chaos experiment will delete pods of the application periodically, while the Iter8 experiment will send requests to the app and verify if it is able to satisfy SLOs.
 
     ![Chaos with SLO Validation](../../images/chaos-slo-validation.png)
 
 ??? warning "Setup Kubernetes cluster and local environment"
-    0. If you completed the [Iter8 getting-started tutorial](../../getting-started/first-experiment.md) (highly recommended), you may skip to step number 6.
+    0. Complete the [getting started tutorial](../../getting-started/first-experiment.md) (highly recommended), and skip to step 7 below.
     1. Setup [Kubernetes cluster](../../getting-started/setup-for-tutorials.md#local-kubernetes-cluster)
     2. [Install Iter8 in Kubernetes cluster](../../getting-started/install.md)
```

Comment:
"[Install Iter8 in Kubernetes cluster](../../getting-started/install.md)" is a hyperlink whereas only "Litmus" is a hyperlink in "Install [Litmus](https://litmuschaos.io/) in Kubernetes cluster".

IMO, I think the latter fits better with the rest of the list. This needs to be changed in multiple places.